vNext/extensions/applicationinsights-react-js/README.md

```diff
@@ -3,20 +3,86 @@
 [![Build Status](https://travis-ci.org/Microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/Microsoft/ApplicationInsights-JS)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-react-js.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-react-js)
 
+React Plugin for the Application Insights Javascript SDK, enables the following:
+
+- Tracking of router changes
+- React components usage statistics
+
 React Plugin for the Application Insights Javascript SDK
 
-## Build:
-```
-npm install -g grunt-cli
-npm install
-npm run build --silent
-```  
+## Getting Started
+
+Install npm package:
 
-## Run unit tests:
+```bash
+npm install --save @microsoft/applicationinsights-react-js
 ```
-npm run test
+
+## Basic Usage
+
+### npm Setup 
```

Comment:
this heading can be deleted as far as I can tell

Reply:
Context properties can be set with Telemetry Initializers, documentation is already in this readme.
There is no root AppInsights object anymore, users can instantiate AI as much as they want, sample code and app show this, withAITracking now requires the user to send the AI instance because of this